genomedata/_load_seq.py

```diff
@@ -214,6 +220,80 @@ def get_num_seq(filenames):
 
     return res
 
+
+def load_chromosome_name_mapping(assembly_report_file):
+    res = {}
+
+    # Assume commented line above the first non-commented line is the header
+    # containing the field names
+    header_line = ""
+
+    # NB: These csv files are assumed to be small enough to be held in memory
+    assembly_report_lines = assembly_report_file.readlines()
+
+    # For every line in the file
+    header_index = 0
+    for line_index, line in enumerate(assembly_report_lines):
+        # If the header line has not been defined
+        if not header_line:
+            # Assume it is the first line
+            header_line = line
+
+        # If the line does not start with a comment
+        if not line.startswith(ASSEMBLY_REPORT_COMMENT_DELIMITER):
+            # Stop and use the previous line as header line
+            break
+        # Otherwise
+        else:
+            # Use the current line as the candidate for the header line
+            header_line = line
+            header_index = line_index
+
+    # Retrive the string after the comment delimiter and split on whitespace
+    # for the headers
+    header_fields = \
+    header_line.split(ASSEMBLY_REPORT_COMMENT_DELIMITER)[1].strip().split()
+
+    # Return a dictionary with header fields for keys containing a list for
+    # their respective column
+    for header in header_fields:
+        res[header] = []
+
+    for row in csv.DictReader(assembly_report_lines[header_index+1:], header_fields,
+                              delimiter=ASSEMBLY_REPORT_FIELD_DELIMITER):
+        for header in header_fields:
+            res[header].append(row[header])
+
+    return res
+
+
+def get_chromosome_name(name, chromosome_name_map, name_style):
+    res = name
```

Comment:
Early return
```
if not chromosome_name_map:
    return name
```

`res` seems unnecessary here, can then replace the other use in this function by use of `return`